package iso3.pt.dao;

public class UserNotFoundException extends Exception {
	public UserNotFoundException()
	{
		System.out.println("Dentro de Exception");
		
	}
	
}
